seedqreader.py

```diff
@@ -367,12 +374,16 @@ class DisplayQR(QThread):
 
     video_stream = Signal(object)
 
-    def __init__(self, parent):
+    def __init__(self, parent, delay):
         QThread.__init__(self)
         self.parent = parent
+        self.set_delay(delay)
         self.qr_data: QRCode | MultiQRCode = None
         self.stop = False
 
+    def set_delay(self, delay):
+        self.delay = delay
+
```

Comment:
i'd prefer we keep simple:
```suggestion
        self.delay=delay
        self.qr_data: QRCode | MultiQRCode = None
        self.stop = False

```